src/hub.js

```diff
@@ -1282,7 +1284,8 @@ document.addEventListener("DOMContentLoaded", async () => {
             entryDisallowed: !hubChannel.canEnterRoom(uiProps.hub)
           });
 
-          const occupantCount = Object.entries(presence.state).length;
+          const userIds = Object.getOwnPropertyNames(presence.state);
```

Comment:
i'd refer to these as `sessionIds`, since in phoenix presence context, they are the `SessionSocket` identifiers.